src/ConnectedVMware/test/Get-AzConnectedVMwareVMTemplate.Tests.ps1

```diff
@@ -1,33 +0,0 @@
-if(($null -eq $TestName) -or ($TestName -contains 'Get-AzConnectedVMwareVMTemplate'))
-{
-  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
-  if (-Not (Test-Path -Path $loadEnvPath)) {
-      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
-  }
-  . ($loadEnvPath)
-  $TestRecordingFile = Join-Path $PSScriptRoot 'Get-AzConnectedVMwareVMTemplate.Recording.json'
-  $currentPath = $PSScriptRoot
-  while(-not $mockingPath) {
-      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
-      $currentPath = Split-Path -Path $currentPath -Parent
-  }
-  . ($mockingPath | Select-Object -First 1).FullName
-}
-
-Describe 'Get-AzConnectedVMwareVMTemplate' {
-    It 'List' -skip {
-        { throw [System.NotImplementedException] } | Should -Not -Throw
```

Comment:
I think we can keep these test ps1 scripts here. Because these files will be generated again after next run for autorest and here is no harm in keeping them

Reply:
Fixed